### ISSUE TITLE
Ubuntu 22.04 as Supported Target

### ISSUE THREAD

**Issue by theofficialgman** [openSUSE/open-build-service#12496]
**Is your feature request related to a problem? Please describe.**
Multiple developers I know are waiting on a 22.04 target to become available on OBS before they can publish software.

**Describe the solution you'd like**
Provide a 22.04 target and an expected date that this will be ready so developers can know.

**Comment by eddy-geek** [comment]
From a cursory look the config is not in this repo.

From the [mailing list](https://lists.opensuse.org/archives/list/buildservice@lists.opensuse.org/thread/WOWJZMKI7H2H5X5UB26GWGYFK64AXMYV/):

> That's already done. It will not be displayed in the distribution list until GA, though.
> If you want to build for Ubuntu 22.04 you have to get to the Distributions page and "add from project". There you can enter xUbuntu22 and start building.

**Comment by abraunegg** [comment]
@eddy-geek , @linuxrrze , @OpenSuSE Team

> That's already done. It will not be displayed in the distribution list until GA, though.
If you want to build for Ubuntu 22.04 you have to get to the Distributions page and "add from project". There you can enter xUbuntu22 and start building.

Please can you check and validate that Ubuntu 22.04 is available on the OneSuSE Build Service. Ubuntu 22.04 went GA on 21st April 2022 - and based on this comment, it should be available to build against.

The option to build against Ubuntu 22.04 is just not available - see: https://github.com/abraunegg/onedrive/issues/1933#issuecomment-1112935979 for details

Please can you look into this.

**Comment by eddy-geek** [comment]
This is done as per mailing-list. (No idea if it will help you @linuxrrze ). This issue can be closed I think,

**Comment by hennevogel** [comment]
I'm sorry that I have to close this, but we are tracking issues in the source code in here. Github issues are ***not*** a way to request support for some specific instance, like the openSUSE on in this case.

However there is a vibrant community of developers, admins and users that support each other in their usage of the OBS. They meet online in a the OBS IRC chat (irc://irc.freenode.net/openSUSE-buildservice) or on the [OBS mailing list](mailto:opensuse-buildservice+subscribe@opensuse.org) ([archive](http://lists.opensuse.org/archive/opensuse-buildservice/)). Here you can ask questions if you have trouble understanding something, you can seek advice and mingle with other OBS community members. 

Thank you for your consideration.
